crates/apollo-compiler/src/database/hir.rs

```diff
@@ -189,8 +189,8 @@ pub struct OperationDefinition {
 
 impl OperationDefinition {
     /// Get a reference to the operation definition's ty.
```

Comment:
a clearer doc comment could also work to separate this type of `ty` from `named_type`
```suggestion
    /// Get the type of the operation: Query, Mutation, or Subscription
```